openrtb_ext/bidders.go

```diff
@@ -187,6 +188,7 @@ var BidderMap = map[string]BidderName{
 	"yieldmo":           BidderYieldmo,
 	"yieldone":          BidderYieldone,
 	"zeroclickfraud":    BidderZeroClickFraud,
+	"between":           BidderBetween,
```

Comment:
Can this entry be moved to its alphabetical order spot above?

Reply:
fixed